internal/server/kong/ws/config/version_compatibility.go

```diff
@@ -41,14 +41,37 @@ const (
 	Route
 	Upstream
 	Vault
+	ConsumerGroup
+	ConsumerGroupConsumer
+	ConsumerGroupPlugin
 )
 
 func (u UpdateType) String() string {
-	return [...]string{"plugin", "plugin", "service", "route", "upstream", "vault"}[u]
+	return [...]string{
+		"plugin",
+		"plugin",
```

Comment:
was there before, but why twice "plugin"?

Reply:
honestly, this is a bit silly, but it's because the first and second `UpdateType`s are `Plugin` and `CorePlugin`, which are represented the same way in the config table in the DP. We should definitely clean up/refactor this folder at some point.

Reply:
ah, thanks for clarification.